packages/apps/transfer/src/components/Global/FormStatusNotification/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { INotificationProps, Notification, SystemIcon } from '@kadena/react-ui';
+
+import { containerStyle } from './styles.css';
+
+import useTranslation from 'next-translate/useTranslation';
+import React, { type FC, useCallback, useEffect, useState } from 'react';
+
+export type FormStatus = 'idle' | 'successful' | 'erroneous' | 'processing';
+
+export type Titles = Record<FormStatus, string>;
+export type Bodies = Record<FormStatus, string>;
+
+const statusToColorMap: Record<FormStatus, INotificationProps['color']> = {
+  erroneous: 'negative',
+  idle: 'warning',
+  processing: 'info',
+  successful: 'positive',
+};
+
+const statusToIconMap: Record<FormStatus, INotificationProps['icon']> = {
+  erroneous: SystemIcon.AlertBox,
+  idle: SystemIcon.AlertCircleOutline,
+  processing: SystemIcon.Information,
+  successful: SystemIcon.Check,
+};
+
+export interface IFormStatusNotificationProps {
+  status?: FormStatus;
+  title?: string;
+  body?: string;
+  statusTitles?: Partial<Titles>;
+  statusBodies?: Partial<Bodies>;
+  children?: INotificationProps['children'];
+}
+
+export const FormStatusNotification: FC<IFormStatusNotificationProps> = (
+  { status, body, title, statusTitles, statusBodies, children } = {
+    status: 'idle',
+  },
+) => {
+  const { t } = useTranslation('common');
+  const [show, setShow] = useState(status !== 'idle');
+
+  useEffect(() => {
+    setShow(status !== 'idle');
+  }, [status]);
+
+  const titles: Titles = {
+    ...{
+      idle: t('form-status-title-idle'),
```

Comment:
we could consider using nested translation to slightly reduce code duplication by being able to use `t('form-status-title', {status: 'successful'})` here

Reply:
Ah yes, good suggestion. But depends on which approach we're taking with the translation keys. If we're sticking to what we're doing now, I guess it doesn't make much sense perhaps..

Reply:
Will be refactored later, in a separate PR. Created a ticket for this; https://app.asana.com/0/1204958723576385/1205129264294989/f